src/expressions/loop-expr.md

```diff
@@ -44,10 +67,15 @@ while i < 10 {
 
 ## `while let` loops
 
+> **<sup>Syntax</sup>**  
+> [_WhileLetExpression_] :  
+> &nbsp;&nbsp; `while` `let` _Pattern_ `=` [_Expression_]<sub>except struct expression</sub>
```

Comment:
All of the other loop expressions are named after what they do. So shouldn't this one be called `PredicatePatternLoopExpression`?

Reply:
Done